Source/WebKit/WebProcess/Plugins/PDF/UnifiedPDF/UnifiedPDFPlugin.h

```diff
@@ -30,15 +30,30 @@
 #include "PDFDocumentLayout.h"
 #include "PDFPluginBase.h"
 #include <WebCore/GraphicsLayer.h>
+#include <wtf/OptionSet.h>
 
 namespace WebKit {
-
 class WebFrame;
+class WebMouseEvent;
 
 class UnifiedPDFPlugin final : public PDFPluginBase, public WebCore::GraphicsLayerClient {
 public:
     static Ref<UnifiedPDFPlugin> create(WebCore::HTMLPlugInElement&);
 
+    enum class ElementUnderMouse : uint16_t {
+        None = 1 << 0,
```

Comment:
Why is None bit 1 instead of the lack of bits?

Reply:
If this is used in an OptionSet, there should be no zero value.

Cool kids are aligning the `=` these days (making it easier to see mistakes in the values).

Reply:
None bit 1 was an oversight, fixed. Also aligned the `=` to become a cool kid.